src/main/kotlin/com/cognifide/gradle/aem/pkg/ComposeTask.kt

```diff
@@ -212,6 +212,7 @@ open class ComposeTask : Zip(), AemTask {
         includeVault(project.relativePath(config.vaultProfilePath + "/" + profileName))
     }
 
+    // TODO Inherit dependsOn from aemCompose task dependencies from project that is being included, instead such constant declaration
     fun dependProject(project: Project) {
         dependsOn("${project.path}:${LifecycleBasePlugin.CLEAN_TASK_NAME}")
```

Comment:
if we includeBundles then we could ensure dependsOn `clean` // @piotr-wilczynski  reported that it helps.
while includeContent , clean is probably not required